### ISSUE TITLE
Implement GUI for steering wheel in Android Studio

### ISSUE THREAD

**Issue by stefanieschreiner** [amosproj/amos2022ws02-automotive-test-app#21]
## User story
1.	As a User
2.	I want to be able to enter an area in the test app where I can see a multifunctional steering wheel 
3.	So that I can get an overview of which button simulations are available. 

With this User story, the graphical user interface of the steering wheel is to be implemented in Android Studio.
The design has been created with #37 

The following buttons are needed (the list might be extended with upcoming requirements):
* SkipForward
* SeekForward (natively available from Android)
* Play/Pause
* Voicecontrol
* Phone (Event „HeadsetHook()")

## Acceptance criteria
* The graphical user interface of the steering wheel is implemented.
* An XML for the GUI of the steering wheel has been created in Android studio.

## Notes
Inspiration for GUI design: 
* https://www.thedrive.com/content-b/message-editor/1623169499432-m163657-2.jpg?quality=60
* https://www.bedienungsanleitu.ng/audi/a4-2021/anleitung?p=15
* https://www.bedienungsanleitu.ng/audi/a4-2021/anleitung?p=31

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by i315315** [comment]
Reopened as issue has been automatically closed with a PR.
